.travis.yml

```diff
@@ -148,6 +152,17 @@ script:
         $PYTHON --version && $PYTHON -c 'import neuron; neuron.test()';
       fi;
     fi;
+  - if [ "$NRN_ENABLE_MPI_DYNAMIC" == "ON" ]; then
+      neurondemo -c 'demo(4)' -c 'run()' -c 'quit()';
+      python -c "import neuron; neuron.test(); neuron.test_rxd()";
+      nrniv -c "print \"hello\"";
+      rm -rf x86_64;
+      nrnivmodl .;
+      ./x86_64/special -python -c "import neuron; neuron.test(); neuron.test_rxd(); quit()";
+      ./x86_64/special -c "print \"hello\"";
+      mpirun -n 2 python src/parallel/test0.py -mpi;
+      mpirun -n 2 nrniv -python src/parallel/test0.py -mpi;
```

Comment:
@iomaganaris : here we have to check if this really prints two distinct ranks i.e. 0 and 1

Reply:
just a suggestion: we can do:
* pass an extra argument (e.g. 2) to `test0.py`
* update `test0.py` so that if an extra arg is there, it treat that as a total host and it should be equal to `nhost`. Otherwise abort with an error.

Reply:
I tried adding a simple optional parameter `--expected-hosts` to test this. I didn't want to complicate more than that the python script.
Let me know if it's okay